tools/content-generator/src/main/java/org/projectnessie/tools/contentgenerator/cli/ContentGenerator.java

```diff
@@ -35,5 +43,40 @@
     })
 public abstract class ContentGenerator<API extends NessieApiV2> {
 
-  public abstract API createNessieApiInstance();
+  @CommandLine.Option(
+      names = {"-u", "--uri"},
+      scope = CommandLine.ScopeType.INHERIT,
+      description = "Nessie API endpoint URI, defaults to http://localhost:19120/api/v2.")
+  private URI uri = URI.create("http://localhost:19120/api/v2");
+
+  @CommandLine.Option(
+      names = "--nessie-client",
+      description = "Name of the Nessie client to use, defaults to HTTP suitable for REST.")
+  private String nessieClientName;
```

Comment:
Why bother when we can do `--nessie-option nessie.client-builder-name=ABC`?.. I guess in most cases it will not have to be overwritten anyway.

Reply:
We already have a "--nessie-api" parameter in Nessie-GC, just copied the existing approach.